processor/transformprocessor/README.md

```diff
@@ -22,6 +22,8 @@ it references an unset map value, there will be no action.
 - `keep_keys(target, string...)` - `target` is a path expression to a map type field. The map will be mutated to only contain
 the fields specified by the list of strings. e.g., `keep_keys(attributes, "http.method")`, `keep_keys(attributes, "http.method", "http.route")`
 
+- `truncateAll(target, limit)` - `target` is a path expression to a map type field. `limit` is an integer.  The map will be mutated such that all string values are truncated to the limit. e.g., `truncate(attributes, 100)` will truncate all string values in `attributes` such that all string values have less than or equal to 100 characters.  Non-string values are ignored.
```

Comment:
I'd suggest this be renamed to `truncate_all` or alternatively have `keep_keys` renamed to `keepKeys` for consistency

Reply:
You're right, should be truncate_all